File: app/src/main/java/vergecurrency/vergewallet/service/model/wallet/BNSendResponse.java
package vergecurrency.vergewallet.service.model.wallet;

import com.google.gson.Gson;

public class BNSendResponse {

	private String txid;

	public static BNSendResponse decode(String JSON) {
		return new Gson().fromJson(JSON, BNSendResponse.class);
	}

	public String getTxid() {
		return txid;
	}

	public void setTxid(String txid) {
		this.txid = txid;
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/service/model/wallet/BNSendResponse.kt
package vergecurrency.vergewallet.service.model.wallet

import com.google.gson.Gson

class BNSendResponse {

    var txid: String? = null

    companion object {

        fun decode(JSON: String): BNSendResponse {
            return Gson().fromJson(JSON, BNSendResponse::class.java)
        }
    }
}
